bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,18 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = ''
+            for line_nr in sorted(diff._changes):
```

Comment:
there's probably another member that allows you to iterate over the changes :+1: